.github/workflows/npm.yml

```diff
@@ -117,17 +117,17 @@ jobs:
       - run: mkdir -p npm/workerd/lib
       - run: mkdir -p npm/workerd/bin
       - name: Build node-install
-        run: npx esbuild npm/lib/node-install.ts --outfile=npm/workerd/install.js --bundle --target=node16 --define:LATEST_COMPATIBILITY_DATE="\"${LATEST_COMPATIBILITY_DATE}\"" --define:WORKERD_VERSION="\"${WORKERD_VERSION}\"" --platform=node --external:workerd --log-level=warning
+        run: npx esbuild npm/lib/node-install.ts --outfile=npm/workerd/install.js --bundle --target=node18 --define:LATEST_COMPATIBILITY_DATE="\"${LATEST_COMPATIBILITY_DATE}\"" --define:WORKERD_VERSION="\"${WORKERD_VERSION}\"" --platform=node --external:workerd --log-level=warning
         env:
           WORKERD_VERSION: ${{ needs.version.outputs.version }}
           LATEST_COMPATIBILITY_DATE: ${{ needs.version.outputs.date }}
       - name: Build node-shim
-        run: npx esbuild npm/lib/node-shim.ts --outfile=npm/workerd/bin/workerd --bundle --target=node16 --define:LATEST_COMPATIBILITY_DATE="\"${LATEST_COMPATIBILITY_DATE}\"" --define:WORKERD_VERSION="\"${WORKERD_VERSION}\"" --platform=node --external:workerd --log-level=warning
+        run: npx esbuild npm/lib/node-shim.ts --outfile=npm/workerd/bin/workerd --bundle --target=node18 --define:LATEST_COMPATIBILITY_DATE="\"${LATEST_COMPATIBILITY_DATE}\"" --define:WORKERD_VERSION="\"${WORKERD_VERSION}\"" --platform=node --external:workerd --log-level=warning
```

Comment:
What's the reason for not targeting LTS? node18 active support ended, and even node 20(LTS) will end in 1 month. 

Ref: https://endoflife.date/nodejs

Reply:
My thinking was that we should target the oldest version developers would reasonably be using, I'm not sure how quickly folks update to the latest version in the Node.js ecosystem but as long as 18 is being supported we should target since it (presumably) costs us nothing for this small application

Reply:
It's hard to distinguish when a technology is not used. Are you thinking about EOL change to drop node18?

Reply:
Actually looked at the generated output – it's identical across node versions which is unsurprising since there's not a whole lot of code to begin with. The esbuild version only has cosmetic differences between 0.15 and 0.23, so it is inconsequential too. I think this can go ahead as-is, the node18 label is least likely to cause confusion IMHO.